src/pages/variantEntity/TableSummaryKfStudies.module.scss

```diff
@@ -0,0 +1,11 @@
+@import 'src/style/themes/default/colors';
+
+.participantNumLink {
+  color: $primary-color;
+  text-decoration: underline;
+  font-weight: bold;
+}
```

Comment:
can you provide an explanation why this button link is different that the others in the other table and other pages?

Reply:
check in the current code (before the pr) that the style was from a completely different page. In that sense, it's a (small) improvement. 
If I remove that style I do not obtain what is visually desired.

So, in my opinion it s better than what it were prior to the pr. However, it is far from being ok. But we have another ticket that take care of all the links style-wise. At that point, that file could be removed.

Reply:
But it added technical debt and work to the person who will do the task next. Better to close your PR fast and take the next Issue and fix it correctly.